### PR TITLE
317 cypress navigation

### PR DESCRIPTION
**Pull request by philiphand**

### CODE REVIEW

**Comment by philiphand** [comment]
Alt klart her nå! @fellmirr

File: cypress/e2e/navigation.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            cy.get("a").click({ force: true })
+        })
+        cy.url().should('include', '/articles')
+    });
+
+    it("Tests newsletter signup", () => {
+        cy.get("[data-cy=newsletter-input]").scrollIntoView()
+        cy.get("[data-cy=newsletter-input]").type("test@gieffektivt.no")
+        cy.get('[data-cy=newsletter-input]').should('have.value', 'test@gieffektivt.no')
+        cy.get("[data-cy=newsletter-submit]").click()
```
Comment (fellmirr) [inline cypress/e2e/navigation.js:137]: This should be stubbed right, so we don't make a whole bunch of signups? :P

**Review by fellmirr — CHANGES_REQUESTED** [review]
Good stuff, if we add a stub for newsletter signups this looks very promising :)

**Review by fellmirr — APPROVED** [review]
:+1:

**Pull request merged**